Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php

```diff
@@ -115,9 +115,11 @@ private function fixScalars(Line $line, $tag)
      */
     private static function normalizeTypes(array $types)
     {
-        foreach ($types as $index => $type) {
-            $types[$index] = self::normalizeType($type);
+        foreach ($types as $index => &$type) {
+            $type = self::normalizeType($type);
```

Comment:
switching to a reference for ``$type`` is causing a reference mismatch when calling ``normalizeType``. So the previous code was actually better. don't try to be smarter than PHP regarding memory management. In many cases, you will actually make things worse rather than better.
See http://jpauli.github.io/2014/06/27/references-mismatch.html for the explanation about reference mismatches